addon-test-support/index.js

```diff
@@ -0,0 +1,15 @@
+const SINON = self.sinon;
+
+export function createSandbox() {
+  QUnit.config.current.testEnvironment.sandbox = self.sinon = SINON.sandbox.create();
```

Comment:
Should be importing QUnit in this file  I think

Reply:
Agreed.